vllm/attention/ops/hpu_paged_attn.py

```diff
@@ -61,6 +61,8 @@ def write_to_paged_cache(key: torch.Tensor, value: torch.Tensor,
 
     @staticmethod
     def forward_decode(**kwargs) -> torch.Tensor:
+        if kwargs.get("kv_lora_rank", None):
+            return ops.flat_pa_mla(**kwargs)
```

Comment:
we have update this in vllm-hpu-extension? if so, no need to update its version?